modules/custom-tuning-specification.adoc

```diff
@@ -121,3 +156,50 @@ catch-all to set `openshift-node` profile, if no other profile with higher
 priority matches on a given node.
 
 image::node-tuning-operator-workflow-revised.png[Decision workflow]
+
+.Example: MachineConfigPool based matching
+[source, yaml]
+----
+apiVersion: tuned.openshift.io/v1
+kind: Tuned
+metadata:
+  name: openshift-node-custom
+  namespace: openshift-cluster-node-tuning-operator
+spec:
+  profile:
+  - data: |
+      [main]
+      summary=Custom OpenShift node profile with an additional kernel parameter
+      include=openshift-node
+      [bootloader]
+      cmdline_openshift_node_custom=+skew_tick=1
+    name: openshift-node-custom
+
+  recommend:
+  - machineConfigLabels:
+      machineconfiguration.openshift.io/role: "worker-custom"
+    priority: 20
+    profile: openshift-node-custom
+----
+
+To minimize Node reboots, it is recommended to label the target Nodes with
```

Comment:
s/Node/node
s/it is recommended to label the target/label the target

Reply:
Good catch!